ui/lib/config-ui/addon/routes/login-settings/index.js

```diff
@@ -4,5 +4,26 @@
  */
 
 import Route from '@ember/routing/route';
+import { service } from '@ember/service';
 
-export default class LoginSettingsRoute extends Route {}
+export default class LoginSettingsRoute extends Route {
+  @service api;
+  @service capabilities;
+  @service store;
+
+  async model() {
+    const adapter = this.store.adapterFor('application');
+
+    const { data } = await adapter.ajax('/v1/sys/config/ui/login/default-auth', 'GET', {
+      data: { list: true },
+    });
```

Comment:
we'll hopefully get to replace this with the api service when the backend API is built! But this looks great for now